tests/scipy_signal_test.py

```diff
@@ -49,10 +50,12 @@ class LaxBackedScipySignalTests(jtu.JaxTestCase):
       for mode in ['full', 'same', 'valid']
       for op in ['convolve', 'correlate']
       for dtype in default_dtypes
-      for xshape in onedim_shapes
-      for yshape in onedim_shapes))
+      for xshape in onedim_shapes + twodim_shapes + threedim_shapes
+      for yshape in onedim_shapes + twodim_shapes + threedim_shapes))
   def testConvolutions(self, xshape, yshape, dtype, mode, jsp_op, osp_op):
     rng = jtu.rand_default(self.rng())
+    if xshape != yshape:
+      return
```

Comment:
This seems strange to only test cases where the two arrays have equal shape. Maybe instead in the parametrization we can do something like:
```python
for shapeset in [onedim_shapes, twodim_shapes, threedim_shapes]
for xshape in shapeset
for yshape in shapeset
```
Then tests will cover mixed pairs of shapes with the same dimension.

Reply:
Makes sense, will do!